src/Context/DrupalErrorCheckApiContext.php

```diff
@@ -53,7 +53,7 @@ public static function checkForWatchdogErrors() {
     $query = \Drupal::database()->select('watchdog', 'w');
     $query->fields("w");
     $query->condition('timestamp', $timestamp, '>');
-    $query->condition('severity', RfcLogLevel::WARNING, '<=');
```

Comment:
That's great, but might break existing test suites. We must be more cautious here.
Could you add an optional behat parameter for that, so one can configure it in behat.yml and keep defaulting to warning? see http://behat.org/en/latest/user_guide/context.html#context-parameters
Obviously, that configuration needs to be documented at the README.